profile.html

```diff
@@ -26,55 +32,74 @@
 
-        <img class="profilePage_clock" src="images\profilecassette.png" alt="clock">
-
-
-
-        <div class="profilePage_pastContent">
+            <div class="profilePage_title">
+                <h1>Profile</h1>
+            </div>
+      </header>
 
-            <p>home for past and saved content</p>
+        <!--<img class="profilePage_clock" src="images\profilecassette.png" alt="clock">-->
 
-        </div>
+        <div class="main-content">
 
-        <div class="profilePage_createContent">
+          <div class="profile-bio">
+            <div class="bio-card">
+                <img src="img.jpg" alt="John" style="width:100%">
+                <h1>John Doe</h1>
+                <p class="bio-username">johndoeisamazing</p>
+                <p class="bio-location">Blah city, United States of Blah</p>
+                <p><button class="bio-button">Follow</button></p>
+            </div>
+          </div>
 
-            <p>this is the contianer for the input of the posts</p>
-            <i class="fa fa-user-circle-o profilePage_userProfileIcon" style="font-size:36px"></i>
+          <div class = "profile-content">
+            <div class="profile-create">
+              <h3> Create new post </h3>
 
-            <div class="profilePage_inputField">
-                <textarea name="" id="" cols="55" rows="5">
-                    Start Typing here or Upload your story
-                </textarea>
+                <div class="post-form">
+                    <form action="#" name="createPostForm" method="post">
+                        <textarea id="createPostText" placeholder="Start typing..." class="text" cols="53" rows ="10" name="createPostText"></textarea>
+                        <input type="submit" value="Create Post" name="profile-submit" id="profile-submit" class="submitButton">
+                    </form>
+                </div>
             </div>
 
-        </div>
+            <div class="profile-tab">
+                <ul class="nav nav-tabs">
+                  <li class="active"><a data-toggle="tab" href="#past-posts">Past Posts</a></li>
+                  <li><a data-toggle="tab" href="#saved-posts">Saved Posts</a></li>
+                </ul>
+
+                <div class="tab-content">
+                    <div id="past-posts" class="tab-pane fade in active">
+                        <h3>Past posts</h3>
+                        <div id="past-posts-id">
+                            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.jkenrfbgebqgjhbwgbbb fn de grw nkjer jrejkgre jregk;af hfke kegr vkebe nvken hkjen e eregk jkgrhgre jkeh e ek jjer jhrgre ehejrner hwfkfwkfb fbwbfr rf kbf fi kbfr bf n jrbfjrfr fbj jbf rbfrk kbfjr nr nrfn r hrbgre jrnereb n fkrbfr frbgkre fn nbklebf wnvje jrevew vrn hre ejvhre kr krv e</p>
+                        </div>
+                    </div>
+
+                    <div id="saved-posts" class="tab-pane fade">
```

Comment:
Is there a particular reason you are using the id's here? maybe for javascript purposes? if not, you can add a second class name and use that for styling purposes.

Reply:
removed